REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,30 @@
+# New Submissions
+
+## Info section/style metadata
+- Style has documentation link (click on the link to make sure it's actually the new documentation rather than the link left in from the template)
+  - Where no public documentation is available, we offer people to upload it to github (though be mindful of copyright) and then link to the github post. We rarely make exceptions to this requirement
+- Style has template link _or_ submitter credibly states that they started from scratch (you can tell by macro names & structure, typically)
+- Journal styles 
```

Comment:
This should be a header

Reply:
I've made that change. I've also added a header for the comment on locales.